content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst

```diff
@@ -0,0 +1,91 @@
+===============================
+Manage Warehouses and Locations
+===============================
+
+Terminology
+===========
+
+Warehouse
+---------
+
+In Odoo, a **Warehouse** is the actual building/place in which your items are stocked. You can
+setup multiple warehouses and create moves between warehouses.
+
+Location
+--------
+
+A **Location**, is a specific space within your warehouse. It can be considered as a sublocation of
+your warehouse, as a shelf, a floor, an aisle, etc. Therefore, a location is part of one warehouse
+only and it is not possible to link one location to multiple warehouses. You can configure as many
+locations as you need under one warehouse.
```

Comment:
```suggestion
A **Location** is a specific space within the warehouse. It can be a sublocation of the warehouse
(a shelf, a floor, an aisle, and so on). Therefore, a location is part of one warehouse only and it
is not possible to link one location to multiple warehouses. In Odoo, as many locations can be
configured as needed under one warehouse.
```

Left the bold formatting on `Location` instead of changing it to a `:guilabel:` because here, `Location` is being introduced as a term, not a GUI element that the reader must interact with.